src/tools/dashboard/src/litani.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//! Utilities to interact with the `Litani` build accumulator.
+
+use pulldown_cmark::escape::StrWrite;
+use std::collections::HashMap;
+use std::path::Path;
+use std::process::{Child, Command};
+
+/// Data structure representing a `Litani` build.
+pub struct Litani {
+    /// A buffer of the `spawn`ed Litani jobs so far. `Litani` takes some time
+    /// to execute each `add-job` command and executing thousands of them
+    /// sequentially takes a considerable amount of time. To speed up the
+    /// execution of those commands, we spawn those commands sequentially (as
+    /// normal). However, instead of `wait`ing for each process to terminate,
+    /// we add its handle to a buffer of the `spawn`ed processes and continue
+    /// with our program. Once we are done adding jobs, we wait for all of them
+    /// to terminate before we run the `run-build` command.
```

Comment:
We can just say that we spawn all litani asynchronously and wait for them to terminate.